contracts/Lock.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+// Uncomment this line to use console.log
+// import "hardhat/console.sol";
+
+contract Lock {
```

Comment:
Could you explain what this contract is doing? and how it works with `AckToEarn`?

Reply:
This doesn't do anything and is unrelated to our project... this is the default contract that comes with the Hardhat setup... I plan to remove it later but it's here for now as a guide.

Reply:
gotcha 👍